contrib/prometheus/prometheus-all.json

```diff
@@ -72,11 +86,25 @@
                         ]
                     },
                     {
-                        "name": "kubectl",
-                        "image": "gcr.io/google_containers/kubectl:v0.18.0-350-gfb3305edcf6c1a",
-                        "args": [
-                            "proxy", "-p", "8001", "--api-prefix=/"
+                        "image": "jayunit100/prompush:0.2.0",
```

Comment:
happy to do so.  Can we do in a follow on PR ? The reason being that there are other images i'd like to add to Gcr and i don't know who will / how to  push them, so its a independent effort.